src/styles/clock.scss

```diff
@@ -0,0 +1,31 @@
+.clock {
+  position: relative;
+  top: 50%;
+  left: 50%;
+
+  &__sec {
+    @include positionClocks();
+    background-color: #c25d5d;
+
+    animation-name: moveRound;
+    animation-duration: 1s;
+    @include animationSet();
+  }
```

Comment:
As a variant, you can include these two properties in your mixin, so it would take duration as a parameter
f.e. @include animation(1s)

Reply:
Good point